woocommerce/woocommerce-ios
# Upgrade Tracks

### Description

This new version of Tracks should address a limitation introduced when upgrading to Sentry v7 which resulted in manual crashes triggered due to inconsistent Core Data state to lack release metadata.

### Testing instructions
There's nothing really to test here, as the library does all the heavy lifting. Checkout internal ref. paaHJt-3Jt-p2#comment-6773 for details on the approach I took to verify the events on Sentry.

---
